linkerd/docs/namer.md

```diff
@@ -354,6 +355,7 @@ host | `marathon.mesos` | The Marathon master host.
 port | `80` | The Marathon master port.
 uriPrefix | none | The Marathon API prefix. This prefix depends on your Marathon configuration. For example, running Marathon locally, the API is avaiable at `localhost:8080/v2/`, while the default setup on AWS/DCOS is `$(dcos config show core.dcos_url)/marathon/v2/apps`.
 ttlMs | `5000` | The polling interval in milliseconds against the marathon API.
+useHealthCheck | `false` | If `true`, rely on Marathon health checks. If `false`, rely on linkerd's built-in resiliency algorithms.
```

Comment:
What about something like:

If `true`, exclude app instances that are failing Marathon health checks.  Even if `false`, linkerd's built-in resiliency algorithms will still apply.